main.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"bufio"
+	"database/sql"
+	"os"
+
+	"github.com/emNakamoto/help_me_language/secrets"
+
+	_ "github.com/go-sql-driver/mysql"
+)
+
+var db *sql.DB
+
+/*
+ * Dictionary... - a very simple struct
+ */
+type Word struct {
+	Language string `json:"language"`
+	Word     string `json:"word"`
+	Meaning  string `json:"meaning"`
+	Tag      string `json:"tag"`
+}
+
+func connectDb(username string, password string) (*sql.DB, error) {
```

Comment:
i would like to put db related functions into db.go and just have main.go be as clean as possible